rdf-delta-server-extra/src/main/java/org/seaborne/delta/server/s3/PatchStorageS3.java

```diff
@@ -120,5 +120,7 @@ public RDFPatch fetch(Id key) {
     }
 
     @Override
-    public void delete(Id id) {}
+    public void delete(Id id) {
+        client.deleteObject(new DeleteObjectRequest(bucketName, idToKey(id)));
```

Comment:
Implemented delete so that an attempt to delete a patch file can be made if an error occurs while attempting to create a log index entry.